chitterApi.test.js

```diff
@@ -0,0 +1,88 @@
+const ChitterApi = require("./chitterApi");
+
+require("jest-fetch-mock").enableMocks();
+
+let api,
+  testPeepsArray,
+  testUserSessionResponse,
+  testPostPeepResponse,
+  testDeletePeepResponse;
+
+describe("ChitterApi class", () => {
+  beforeEach(() => {
+    api = new ChitterApi();
+    testPeepsArray = require("./testPeepsArray");
+    testUserSessionResponse = require("./testUserSessionResponse");
+    testPostPeepResponse = require("./testPostPeepResponse");
+    testDeletePeepResponse = require("./testDeletePeepResponse");
+    fetch.resetMocks();
```

Comment:
Overall the tests are logical and beautifully written. Main question is:

Would the tests have achieved the same objectives with 'simple' data rather than the 'real' examples you've used in each of the files?

You could have all of your test data in one json file (test-data.json for example?)

I like that you've used the example data - but if the tests have the same integrity with simplified data then it would be best to simplify.